global.json

```diff
@@ -1,7 +1,7 @@
 {
   "sdk": {
-    "version": "6.0.100",
-    "rollForward": "latestFeature"
+    "version": "8.0.0",
```

Comment:
The SDK version should be `8.0.100`.

Reply:
Does this change anything, given it's `latestFeature`?

Reply:
`latestFeature` should mean that all `8.0.xxx` versions are valid.
The concern is that `8.0.0` is not valid SDK version number format. I've not found the `global.json` parsing code, but maybe something is depending on the format.